.settings.xml

```diff
@@ -25,15 +25,15 @@
                 <repository>
                     <id>spring-snapshots</id>
                     <name>Spring Snapshots</name>
-                    <url>https://repo.spring.io/snapshot</url>
+                    <url>https://repo.spring.io/libs-snapshot</url>
```

Comment:
Why is that?
The recommendation is really to use just `snapshot` where we have only snapshots.
Mostly artifacts must be resolved from Maven Central.
The artifact under the question is here: https://mvnrepository.com/artifact/io.spring.docresources/spring-doc-resources/0.2.5
Right, it is not in a Central by itself, but in the https://repo.spring.io/release/io/spring/docresources/spring-doc-resources/0.2.5/

So, we probably need to add that https://repo.spring.io/release entry instead of this `libs` mirror.

Reply:
libs-snapshot includes spring milestones and releases.

Reply:
Correct, but I believe it requires an authentication.
We should do like this instead:
```
        maven { url 'https://repo.spring.io/release' }
        maven { url 'https://repo.spring.io/milestone' }
        maven { url 'https://repo.spring.io/snapshot' }
```

Reply:
@corneil I thought we only needed to use the "uber" virtual repos due to JFrog plugin which we only use in SCDF.